### ISSUE TITLE
TypeError: string argument without an encoding  

### ISSUE THREAD

**Issue by mbi** [digitalvalley-nl/django-mailersend#1]
When attaching a MIMEImage, sending fails

![image](https://user-images.githubusercontent.com/131808/222435716-38803fb8-e5c3-41d4-bf54-249da35e0a38.png)

Test case:
```python

from django.core.mail.message import EmailMessage
from email.mime.image import MIMEImage


with open('some-image.jpg', 'rb') as fp:
     data = fp.read()

mime_img = MIMEImage(data)

email_message = EmailMessage(to=['test@test.com'], subject='Test', body='Body')
email_message.attach(mime_img)
email_message.send()
```

**Comment by davidconde-nl** [comment]
Thanks for your report. I will investigate the issue.

**Comment by davidconde-nl** [comment]
Fixed in `v0.2.0-alpha`.

**Comment by mbi** [comment]
Awesome, thanks for the quick fix! :+1:
